sql/catalyst/src/test/scala/org/apache/spark/sql/RowTest.scala

```diff
@@ -29,8 +32,8 @@ class RowTest extends AnyFunSpec with Matchers {
 
   val schema = StructType(
     StructField("col1", StringType) ::
-    StructField("col2", StringType) ::
-    StructField("col3", IntegerType) :: Nil)
+      StructField("col2", StringType) ::
```

Comment:
Scalafmt formatted the file, should I revert the changes related to formatting the code ?

Reply:
Yeah revert all the formatting only changes please

Reply:
I pushed a new commit with the revert of those changes